### DISCUSSION TITLE
Build Fails for windows_arm64 on Mac

### DISCUSSION THREAD

**Discussion by mcolburn** [goreleaser/goreleaser#2495] [Q&A] [answered]
I installed goreleaser using

`brew install goreleaser/tap/goreleaser`

After initializing goreleaser in my go app directory, I ran:

`goreleaser release --snapshot --skip-publish --rm-dist`

It successfully built many target binaries, but failed on windows_arm64:

<img width="1620" alt="Screen Shot 2021-09-16 at 4 29 49 PM" src="https://user-images.githubusercontent.com/1334217/133701408-c7483a65-512f-485a-85f6-d7e954a5deb7.png">

My OS version is: Big Sur 11.6
My go version is: go1.17 darwin/amd64
My goreleaser version is: 0.179.0

Any ideas how to resolve this issue?

I can successfully create the windows_arm64 binary using:

`GOOS=windows GOARCH=arm go build`

Thanks!

**Comment by caarlos0** [comment, marked as answer]
Does `GOOS=windows GOARCH=arm64 go build .` works? 

It seems to me that the version of x/sys you are using does not support windows/arm64 🤔

**Reply by mcolburn** [reply]
Yes, `GOOS=windows GOARCH=arm go build .` works without error.

**Reply by caarlos0** [reply]
hmm, maybe run goreleaser with `--debug`, see which command its trying to run, that might get you more info...

the error itself is on some dependency you're using, so I'm not sure how to help without more context

**Reply by caarlos0** [reply]
marking as right answer since due to lack of response...
